src/engine/renderer/glsl_source/computeLight_fp.glsl

```diff
@@ -24,273 +24,253 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #define COMPUTELIGHT_GLSL
 
 #if !defined(USE_BSP_SURFACE)
-	#define USE_MODEL_SURFACE
+    #define USE_MODEL_SURFACE
```

Comment:
It looks like you're actually rewriting the whole file so I prefer if you use this opportunity to switch to tabs.

Reply:
Hmm, I thought I did replace it with tabs. Maybe I changed some setting in my IDE that converted it into spaces, I'll recheck this later.

Reply:
Fixed.